app/views/rubric_criteria/_rubric_criterion_levels.html.erb

```diff
@@ -1,11 +1,25 @@
+<% content_for :head do %>
```

Comment:
This is interesting; I don't think you should need to include this separate, because the file should already be loaded inside `views/criteria/index.html.erb`.